site/content/docs/commands/storage-init.en.md

```diff
@@ -24,31 +46,51 @@ DynamoDB Flags
                                Must be of the format '<keyName>:<dataType>'.
       --sort-key string        Optional. Sort key for the DDB table.
                                Must be of the format '<keyName>:<dataType>'.
+
 Aurora Serverless Flags
       --engine string               The database engine used in the cluster.
                                     Must be either "MySQL" or "PostgreSQL".
       --initial-db string           The initial database to create in the cluster.
       --parameter-group string      Optional. The name of the parameter group to associate with the cluster.
-      --serverless-version string   Optional. Aurora Serverless version. Must be either "v1" or "v2". (default "v2")
+      --serverless-version string   Optional. Aurora Serverless version.
+                                    Must be either "v1" or "v2". (default "v2")
```

Comment:
```suggestion
                                    Must be either "v1" or "v2" (default "v2").
```

Reply:
This was actually autogenerated by cobra. It appends the `(default "v2")` at the end of the flag description.

I applied this change anyway! However, when people run `copilot storage init --help` they will still see `Must be either "v1" or "v2". (default "v2")` though, and this is sort of not within our control